README.md

```diff
@@ -51,6 +52,15 @@ This work is still in progress. See the development plan.
 - [x] Ability to mount other dirs to the task containers
 - [x] Ability to use a task as a step for another task
 - [x] Ability to get ENV, param, etc values from host environment variables or `.env` file
+- [ ] Ability to install as a Snap package
+- [ ] Ability to install as a Deb package
+- [ ] Ability to install as a RPM package
+- [ ] Ability to install as a Brew package
+
+### [`v2.0`](https://github.com/leopardslab/Dunner/milestone/3) 
```

Comment:
Can you change the `https://github.com/leopardslab/Dunner` to `https://github.com/leopardslab/dunner` everywhere in the PR?